### ISSUE TITLE
Release notes missing for v0.33.0

### ISSUE THREAD

**Issue by aphedges** [huggingface/accelerate#2990]
`accelerate` v0.33.0 was published on PyPI close to two weeks ago, but there are no release notes included in the release, <https://github.com/huggingface/accelerate/releases/tag/v0.33.0>. Previous releases include release notes, so the lack of them is surprising. Is this omission intentional?

**Comment by muellerzr** [comment]
@aphedges nope! Just got lost in the sprint for llama 405 and then I happened to get I’ll immediately after 😅 these will go live tommorow!

**Comment by muellerzr** [comment]
This is up now :)

**Comment by aphedges** [comment]
I can confirm it's there. Thank you very much for taking care of it!
